toolbox/tabs/home/HomePanel.js

```diff
@@ -0,0 +1,15 @@
+/*
+ * This file belongs to Hoist, an application development toolkit
+ * developed by Extremely Heavy Industries (www.xh.io | info@xh.io)
+ *
+ * Copyright © 2018 Extremely Heavy Industries Inc.
+ */
+import {Component} from 'react';
+import {hoistComponent} from 'hoist/core/index';
+
+@hoistComponent()
+export class HomePanel extends Component {
+    render() {
+        return null;
```

Comment:
I am happy to help come up with some intro text for this panel, explaining what the Toolbox is about.

It would be nice to have a standard layout for each of the example tabs where we had a section (presumably at the top) for some optional notes about the component. Not looking to re-write the component doc comments again here (would be interesting in the future to somehow extract them automatically, but don't think that's in scope for this project). 

But maybe a sentence or two for some components - e.g. "The Hoist grid component is built with ag-grid and offers several custom extensions for column rendering, column selection, and user preferences."